src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <>
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age !== undefined
+    && (
+    <p className="Person__age">
+        {`I am ${person.age}`}
+    </p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried !== true
+        && <p>I am not married</p> }
```

Comment:
can be less
check other places 
```suggestion
      {!person.isMarried && <p>I am not married</p> }
```